pkg/cloud/alibaba/alibaba.go

```diff
@@ -0,0 +1,38 @@
+package alibaba
+
+import (
+	"embed"
+
+	"github.com/openshift/cluster-cloud-controller-manager-operator/pkg/cloud/common"
+	appsv1 "k8s.io/api/apps/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	//go:embed assets/*
+	aliFS embed.FS
+
+	aliResources []client.Object
+
+	aliSources = []common.ObjectSource{
+		{Object: &appsv1.Deployment{}, Path: "assets/cloud-controller-manager-daemonset.yaml"},
```

Comment:
Just noticed this doesn't match the filename in the folder, needs to be deployment
```suggestion
		{Object: &appsv1.Deployment{}, Path: "assets/cloud-controller-manager-deployment.yaml"},
```